packages/api/src/command/medical/admin/get-hie-enabled-feature-flags-status.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  getDeployments,
+  getConfigurationContent,
+  CxFeatureFlagStatus,
+} from "@metriport/core/external/aws/app-config";
+import { Config } from "../../../shared/config";
+
+export async function getHieEnabledFFStatus(cxId: string): Promise<CxFeatureFlagStatus> {
+  const region = Config.getAWSRegion();
+  const appId = Config.getAppConfigAppId();
+  const configId = Config.getAppConfigConfigId();
+  const envId = Config.getAppConfigEnvironmentId();
+  const deployments = await getDeployments({
+    region,
+    appId,
+    envId,
+  });
+  const deploymentsLatestToEarliest = deployments
+    .sort((a, b) => (a.DeploymentNumber ?? -1) - (b.DeploymentNumber ?? -1))
+    .reverse();
+  const latestDeployment = deploymentsLatestToEarliest[0];
+  if (!latestDeployment) {
```

Comment:
Whats the purpose of getting the latestDeployment? Why couldnt we just use `getCxsWithCWFeatureFlagValue` in the `app-config.ts` file on the api folder?

Reply:
We should always be retrieving the latest deploy

Reply:
Good call. Done.